propy/PyPro.py

```diff
@@ -401,10 +426,10 @@ def GetALL(self):
         res.update(self.GetMoranAuto())
         res.update(self.GetGearyAuto())
         res.update(self.GetCTD())
-        res.update(self.GetPAAC())
-        res.update(self.GetAPAAC())
-        res.update(self.GetSOCN())
-        res.update(self.GetQSO())
+        res.update(self.GetPAAC(lamda=lamda, weight=weight))
+        res.update(self.GetAPAAC(lamda=lamda, weight=weight))
```

Comment:
The default weight of `GetAPAAC` is `0.5`, but of GetPAAC it's `0.05`. Hence this change is not backwards-compatible.

Reply:
Sorry, I missed the different weight parameters. The weights should be the same as in the original functions for backwards-compatibility.